metrics.go

```diff
@@ -117,11 +118,30 @@ func (c *Connection) registerMetrics(subsystem string) error {
 	return nil
 }
 
+func (c *Connection) GetAPIServiceLabelFromPath(path string) string {
+	if strings.HasPrefix(path, "/api/accounts_mgmt") {
+		return "ocm-accounts-service"
+	} else if strings.HasPrefix(path, "/api/clusters_mgmt") {
+		return "ocm-clusters-service"
+	} else if strings.HasPrefix(path, "/api/authorizations") {
+		return "ocm-authorizations-service"
+	} else if strings.HasPrefix(path, "/api/service_logs") {
+		return "ocm-logs-service"
+	} else {
+		pathParts := strings.Split(path, "/")
+		if len(pathParts) > 3 {
+			pathParts = pathParts[:3]
+		}
+		return "ocm-" + strings.Join(pathParts, "/")
```

Comment:
Should this join on `"-"` instead of `"/"`?

Reply:
I feel this really does not matter. It's just in case it's forgotten when a new service is added. by part of its url, we can tell what goes wrong and fix it accordingly.